client-v2/config/webpack.config.prod-prototype.js

```diff
@@ -0,0 +1,16 @@
+import prod from './webpack.config.prod';
+const path = require('path');
+const webpack = require('webpack');
+
+const definePlugin = new webpack.DefinePlugin({
+  'process.env.APP_URL_PREFIX': JSON.stringify('v2-prototype')
```

Comment:
So I guess we're also adding in a `.env` file to get this working here? Would it be more in keeping to put a new config variable in the s3 bucket? Was this approach quicker?

Reply:
Ah I guess this is what you're doing here? https://webpack.js.org/plugins/define-plugin/

Reply:
@ajwl that's correct 👍